### PR TITLE
Mh photogram ajax

### PR DESCRIPTION
**Pull request by merrillhuang**

### CODE REVIEW

File: app/views/follow_requests/_follow_unfollow.html.erb [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
       <% end %>
     <% elsif follow_request.accepted? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request, method: :delete, class: "btn btn-outline-secondary sender_#{follow_request.sender_id}_recipient_#{follow_request.recipient_id}", remote: true do %>
         Un-follow
       <% end %>
     <% end %>
   <% else %>
-    <%= render "follow_requests/form", follow_request: recipient.received_follow_requests.build %>
+    <%= render "follow_requests/form", follow_request: FollowRequest.new(sender_id: sender.id, recipient_id: recipient.id) %>
```
Comment (Ritumka) [inline app/views/follow_requests/_follow_unfollow.html.erb:15]: Good job at assigning sender and recipient id's!

File: app/views/photos/_likes.html.erb [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <% like = current_user.likes.find_by(photo: photo) %>
 
     <% if like %>
-      <%= link_to like, class: "btn btn-link link-underline-dark link-underline link-underline-opacity-0 link-underline-opacity-100-hover", data: { turbo_method: :delete } do %>
+      <%= link_to like, class: "btn btn-link link-underline-dark link-underline link-underline-opacity-0 link-underline-opacity-100-hover", method: :delete, remote: true do %>
```
Comment (Ritumka) [inline app/views/photos/_likes.html.erb:8]: Clear and concise!

**Review by Ritumka — APPROVED** [review]
Good job, Merrill! I like how thoroughly you commit every change. Looks like a clear lesson in how to complete this assignment .

File: app/views/comments/_comment.html.erb [hunk 3]
```diff
@@ -11,11 +11,11 @@
     </div>
     <div>
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted", remote: true do %>
           <i class="fas fa-edit fa-fw"></i>
         <% end %>
 
-        <%= link_to comment, data: { turbo_method: :delete }, class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to comment, method: :delete, class: "btn btn-link btn-sm text-muted", remote: true do %>
```
Comment (hatcheta1) [inline app/views/comments/_comment.html.erb:18]: nice refactoring!

File: app/views/comments/_form.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<li class="list-group-item">
-  <%= form_with(model: comment) do |form| %>
+<li id="<%= dom_id(comment.photo) %>_<%= dom_id(comment) %>_form" class="list-group-item">
```
Comment (hatcheta1) [inline app/views/comments/_form.html.erb:1]: great use of ```dom_id```

File: app/views/comments/edit.js.erb
Comment (hatcheta1) [inline app/views/comments/edit.js.erb:1]: looks great!

File: app/views/likes/create.js.erb
Comment (hatcheta1) [inline app/views/likes/create.js.erb:1]: nice variable use

**Review by hatcheta1 — APPROVED** [review]

File: Gemfile [hunk 5]
```diff
@@ -3,6 +3,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.1"
 
+gem "active_link_to"
+
+gem "pundit"
```
Comment (aldoportillo) [inline Gemfile:8]: good catch but why are you using `active_link_to`?

File: app/views/comments/_form.html.erb (hunk 4 above)
Comment (aldoportillo) [reply app/views/comments/_form.html.erb:1]: ^

**Review by aldoportillo — APPROVED** [review]

File: Gemfile (hunk 5 above)
Comment (merrillhuang) [reply Gemfile:8]: Something in the old starter code for this project was using active_link_to, so I had to add the gem to get the server to start working.

**Pull request merged**